packages/venia-concept/src/components/MiniCart/miniCart.css

```diff
@@ -37,6 +37,19 @@
 
 .body {
     overflow: auto;
+    position: relative;
+}
+
+/*TODO: Here need a class 'indicator'*/
+
+.body > div {
```

Comment:
I'm not sure we need this css anymore after my last commit.

Reply:
@sirugh yes, this style is no longer needed.